src/gate_detect/test/testUtilities/TestUtils.cpp

```diff
@@ -0,0 +1,92 @@
+#include "TestUtils.h"
+#include <iostream>
+
+using namespace std;
+
+void TestUtils::DisplayGateDetected(cv::Mat mat, std::vector<float> floatVec) {
+    cv::namedWindow("Display window",
+                    cv::WINDOW_AUTOSIZE); // Create a window for display.
+
+    cout << "Gate Vector test " << floatVec[0] << "  " << floatVec[1] << "  "
+         << floatVec[2] << "  " << floatVec[3] << "  " << floatVec[4] << "  "
+         << floatVec[5] << "  " << floatVec[6] << "  " << floatVec[7] << "  "
+         << floatVec[8] << "\n";
+
+    int leftPole, rightPole, topPole;
+
+    leftPole =
+    (1855 / 2) -
+    ((sin(floatVec[2]) * floatVec[1] * ((floatVec[1] - 5.134) / -0.002673)) /
+     (.3048 / 4));
+
+    rightPole = ((1855 / 2) - ((sin(floatVec[5]) * floatVec[4] * 4.0 *
+                                ((floatVec[4] - 5.134) / -0.002673)) /
+                               (.3048)));
+
+    topPole = (int) ((1056 / 2) - ((sin(floatVec[8]) * floatVec[7] * 4.0 *
+                                    ((floatVec[7] - 5.134) / -0.002673)) /
+                                   (.3048)));
```

Comment:
There a lot of magic numbers here? We should make them constants so that it's more clear what we're doing. A few comments would also be helpful.

Reply:
A lot of these numbers depend on the interpolation function we're using to define the distance and angle.

Reply:
That's good information to put in a comment:

`we have chosen to use this interpolation function which relies on these named values ...`

And then have all the dependent numbers as named constants so that readers have at least an idea of what's going on.